kuksa-client/README.md

```diff
@@ -27,11 +27,11 @@ After you have installed the kuksa-client package via pip you can run the test c
 $ kuksa-client
 ```
 
-With default CLI arguments, the client will try to connect to a local VISS server e.g. `kuksa-val-server`.
-If you wish to connect to a gRPC server e.g. `kuksa-databroker`, you should instead run:
+With default CLI arguments, the client will try to connect to a local databroker, e.g. a server supporting the `kuksa.val.v1` protocol..
```

Comment:
`Databroker` with capital `D`?
Shall we also mention that it is "without TLS" ( I prefer not to use the term "insecure" as I see that as a bit ambiguous)